sensor/kubernetes/listener/resource_event_handler.go

```diff
@@ -276,7 +276,9 @@ func handle(
 		seenIDs:                    make(map[types.UID]struct{}),
 		missingInitialIDs:          nil,
 	}
-	informer.AddEventHandler(handlerImpl)
+	if _, err := informer.AddEventHandler(handlerImpl); err != nil {
```

Comment:
could not

Reply:
I mean this is a much bigger issue I guess cause then we won't be processing a type of object

Reply:
Changed to `utils.Should`